ci/pipeline.yml

```diff
@@ -32,6 +32,15 @@ resources:
       branch: master
       tag_filter: "*"
 
+  - type: pull-request
+    name: pr
+    icon: source-pull
+    webhook_token: 0gd4XZNL4Y94zYDLql3C
```

Comment:
You need to add a webhook for this job on GitHub. It's done manually, you can copy an existing one for reference